README.md

```diff
@@ -1 +1,131 @@
+ - **Special Feature Opt-Ins**
+   - [`specialFeatureOptIns`](#specialfeatureoptins)
+ - **Publisher**
+   - [`publisherConsents`](#publisherconsents)
+   - [`publisherCustomConsents`](#publishercustomconsents)
+   - [`publisherLegitimateInterest`](#publisherlegitimateinterest)
+   - [`publisherCustomLegitimateInterest`](#publishercustomlegitimateInterest)
+   - [`publisherRestrictions`](#publisherrestrictions)
+     - This Vector is a special [`PurposeRestrictionVector`]() of [`PurposeRestrictions`]()
+
+**Example with `vendorConsents`**
+
+The `vendorConsents` property on the `TCModel` is a [`SortedVector`]().  This example illustrates the methods of a [`SortedVector`](). With the exception of the `publisherRestrictions`, which implements a different type of [`PurposeRestrictionVector`](), all of the above Vectors will have this interface and functionality.
+
+```java
+// Give Vendor ID 24 consent
+SortedVector vendorIds = new SortedVector();
+TCModel tcModel = new TCModel();
+// Sorted vector is a class with reverse sorted SortedSet elemenets  
+vendorIds.getSet().add(24);
+TCModelEnum.valueOf("vendorConsents").setValue(tcModel,vendorIds); // Using the TCModelEnum class to assign it.
+// OR
+// tcModel.setVendorConsents(vendorIds);
+System.out.println(tcModel.getVendorConsents().getSet().contains(24)); // true
+System.out.println(tcModel.getVendorConsents().getSet().first()); // 24 
+//first() returns the Maximum element since set is reversely sorted.
+System.out.println(tcModel.getVendorConsents().getSet().size()); // 1
+
+// remove vendor 24
+tcModel.getVendorConsents().getSet().remove(24)
+System.out.println(tcModel.getVendorConsents().getSet().contains(24)); // false
+System.out.println(tcModel.getVendorConsents().getSet().first()); // 0
+System.out.println(tcModel.getVendorConsents().getSet().size()); // 0
+
+// give a group of vendors consent
+tcModel.getVendorConsents().getSet().add(14);
+tcModel.getVendorConsents().getSet().add(24);
+tcModel.getVendorConsents().getSet().add(100);
+tcModel.getVendorConsents().getSet().add(102);
+System.out.println(tcModel.getVendorConsents().getSet().contains(24)); // true
+System.out.println(tcModel.getVendorConsents().getSet().contains(14)); // true
+System.out.println(tcModel.getVendorConsents().getSet().contains(200)); // false
+System.out.println(tcModel.getVendorConsents().getSet().first()); // 102
+System.out.println(tcModel.getVendorConsents().getSet().size()); // 5
+
+// loop through all ids 1 to maxId (102 loops)
+for(int i=1;i<=tcModel.getVendorConsents().getSet().first();i++) {
+     // check each id for consent
+
+};
+
+// empty everything
+tcModel.getVendorConsents().getSet().clear();
+
+System.out.println(tcModel.getVendorConsents().getSet().contains(24)); // false
+System.out.println(tcModel.getVendorConsents().getSet().first()); // 0
+System.out.println(tcModel.getVendorConsents().getSet().size()); // 0
+```
+
+### Setting Publisher Restrictions
```

Comment:
Let's remove this until we can actually set publisher restrictions, since we don't support encoding

Reply:
Hi Chris,

are there plans for the future to encode the TCString with java? Or is the intention to not encode the TCString with java at all?

Thx 
Antje

Reply:
Yes the plan is to support encoding in the future, just trying to get over the first hurdle!